Dockerfile

```
@@ -0,0 +1,13 @@
+FROM node:20
+
+WORKDIR /src/app
```

Since I will be changing your docker-compose entirely, kindly update this to more closely match what I'll be doing there.

```suggestion
WORKDIR /usr/app
```